lib/bfd.c

```diff
@@ -57,50 +49,6 @@ struct bfd_source_cache {
 };
 SLIST_HEAD(bfd_source_list, bfd_source_cache);
 
-/**
- * Data structure to do the necessary tricks to hide the BFD protocol
```

Comment:
hmm, I sort of liked that this was not "public" - it's a pity to lose the data-hiding just so that a daemon can ask the question: "is this admin-down?" Could we maybe add an api that answers that question, but leaves the session opaque?

Reply:
Good idea, will change it :+1:

Reply:
Fixed, thanks.